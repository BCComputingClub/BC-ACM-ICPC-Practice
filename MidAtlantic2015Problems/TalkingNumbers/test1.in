1
20222143
1000001
0
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
30
32
40
43
50
54
60
65
70
76
80
87
90
98
100
102
112
131
244
355
466
577
688
799
800
908
1000
1001
2012
3078
4199
5200
9999
10000
20002
30012
49876
55430
61000
72009
77777
99999
100000
201000
321000
400005
540065
650123
765056
999999
1000000
2000003
3000019
20000123
32001234
87012000
98123000
65001011
65123011
100000000
120000000
103000000
200400000
300040000
400005000
500000600
600000070
700000008
123456789
123000789
777777777
999999999
-1
