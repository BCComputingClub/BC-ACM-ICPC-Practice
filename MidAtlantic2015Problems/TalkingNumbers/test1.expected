one
twenty million, two hundred and twenty-two thousand, one hundred and forty-three
one million and one
zero
two
three
four
five
six
seven
eight
nine
ten
eleven
twelve
thirteen
fourteen
fifteen
sixteen
seventeen
eighteen
nineteen
twenty
twenty-one
thirty
thirty-two
forty
forty-three
fifty
fifty-four
sixty
sixty-five
seventy
seventy-six
eighty
eighty-seven
ninety
ninety-eight
one hundred
one hundred and two
one hundred and twelve
one hundred and thirty-one
two hundred and forty-four
three hundred and fifty-five
four hundred and sixty-six
five hundred and seventy-seven
six hundred and eighty-eight
seven hundred and ninety-nine
eight hundred
nine hundred and eight
one thousand
one thousand and one
two thousand and twelve
three thousand and seventy-eight
four thousand, one hundred and ninety-nine
five thousand, two hundred
nine thousand, nine hundred and ninety-nine
ten thousand
twenty thousand and two
thirty thousand and twelve
forty-nine thousand, eight hundred and seventy-six
fifty-five thousand, four hundred and thirty
sixty-one thousand
seventy-two thousand and nine
seventy-seven thousand, seven hundred and seventy-seven
ninety-nine thousand, nine hundred and ninety-nine
one hundred thousand
two hundred and one thousand
three hundred and twenty-one thousand
four hundred thousand and five
five hundred and forty thousand and sixty-five
six hundred and fifty thousand, one hundred and twenty-three
seven hundred and sixty-five thousand and fifty-six
nine hundred and ninety-nine thousand, nine hundred and ninety-nine
one million
two million and three
three million and nineteen
twenty million, one hundred and twenty-three
thirty-two million, one thousand, two hundred and thirty-four
eighty-seven million, twelve thousand
ninety-eight million, one hundred and twenty-three thousand
sixty-five million, one thousand and eleven
sixty-five million, one hundred and twenty-three thousand and eleven
one hundred million
one hundred and twenty million
one hundred and three million
two hundred million, four hundred thousand
three hundred million, forty thousand
four hundred million, five thousand
five hundred million, six hundred
six hundred million and seventy
seven hundred million and eight
one hundred and twenty-three million, four hundred and fifty-six thousand, seven hundred and eighty-nine
one hundred and twenty-three million, seven hundred and eighty-nine
seven hundred and seventy-seven million, seven hundred and seventy-seven thousand, seven hundred and seventy-seven
nine hundred and ninety-nine million, nine hundred and ninety-nine thousand, nine hundred and ninety-nine
